armi/reactor/converters/uniformMesh.py

```diff
@@ -732,6 +756,30 @@ def _mapStateFromReactorToOther(
         """
         pass
 
+    @staticmethod
+    def _calculateReactionRates(lib, keff, assem):
+        """
+        Calculates the neutron reaction rates on the given assembly.
+
+        Notes
+        -----
+        If a block in the assembly does not contain any multi-group flux
+        than the reaction rate calculation for this block will be skipped.
+        """
+        for b in assem:
+            print(b)
+            # Checks if the block has a multi-group flux defined and if it
+            # does not then this will skip the reaction rate calculation. This
+            # is captured by the TypeError, due to a `NoneType` divide by float
+            # error.
+            try:
+                b.getMgFlux()
+            except TypeError:
+                continue
+
+            print(b.getMgFlux())
```

Comment:
Did you mean to keep these two `print` statements?

Reply:
Should be removed now.

Reply:
https://github.com/terrapower/armi/pull/919/commits/d30112ac4f8dc981e37d179e7fcb3f959764ce5b